frontend/packages/dev-console/integration-tests/features/pipelines/event_tab_pipeline_run_page_ODC_4314

```diff
@@ -0,0 +1,68 @@
+Feature: Events tab in Pipeline run and Task run details pages
+	As a user, I want to see Events tab in Pipeline run and Task run details pages
```

Comment:
Use standards we have set for spacings

Reply:
I checked in other files, I see no difference